release/crankshaft--0.0.1.sql

```diff
@@ -137,6 +137,33 @@ BEGIN
 END;
 $$
 LANGUAGE plpgsql VOLATILE;
+CREATE OR REPLACE FUNCTION
+  cdb_create_segment (
```

Comment:
This is a leftover from a previous version. Those functions aren't actually in the codebase anymore but I haven't committed a recompiled crankshaft--0.0.1.sql since I last removed them. Will do that now